dds/DCPS/RTPS/Sedp.h

```diff
@@ -232,6 +236,9 @@ class Sedp : public DCPS::EndpointManager<ParticipantData_t> {
     Msg(MsgType mt, DCPS::MessageId id, const DDS::Security::ParticipantGenericMessage* data)
       : type_(mt), id_(id), pgmdata_(data) {}
 #endif
+
+    static OPENDDS_STRING msgTypeToString(const MsgType type);
```

Comment:
I don't think I understand, are you saying I shouldn't put the const in the function declaration?

Reply:
Yes, `const` doesn't belong there.  You can use it in the definition if you'd like but it doesn't impact the caller.

Reply:
I was not aware of that, I assumed it would be part of the function signature, but I guess as long as it's not a reference it doesn't matter since it gets copied regardless.